frontend/internal-packages/configs/eslint/base.js

```diff
@@ -41,6 +43,24 @@ export function createBaseConfig(options = {}) {
         '@typescript-eslint/no-unsafe-member-access': 'error',
         'require-use-server/require-use-server': 'error',
         'no-non-english/no-non-english-characters': 'error',
+        'unicorn/filename-case': ['error', {
+          cases: {
+            camelCase: true,
+            pascalCase: true
+          },
+          ignore: [
+            '^global-error\\.(tsx?)$',
+            '^instrumentation-client\\.(ts)$',
+            '\\.(css\\.d\\.ts)$',
+            '^.*\\.(test|spec)\\.(ts|tsx|js|jsx)$',
+            '^.*\\.config\\.(ts|tsx|js|jsx|mjs)$',
+            '^.*\\.setup\\.(ts|tsx|js|jsx)$',
+            '^\\..*',
+            'README\\.md$',
+            '^.*\\.(sql|sh|md|json|yaml|yml|toml|lock)$',
+            '^.*_.*\\.ts$'
+          ]
```

Comment:
I am now checking to see if this setting is appropriate. Perhaps there is something I can turn off.

Reply:
done: https://github.com/liam-hq/liam/pull/2369/commits/adb3e38f60c64853e9733e085d2ea1a2217ad8c5